packages/react/src/exampleTypes/ExpandableTagsQuery/index.js

```diff
@@ -132,8 +133,8 @@ let ExpandableTagsQuery = ({
               />
             ))(
               _.reject(
-                _.includes(_, _.memoize(_.keys)(node.context.tags)),
-                _.keys(node.context.keywordGenerations)
+                _.includes(_, _.memoize(keysToLower)(node.context.tags)),
```

Comment:
It occurs to me that the memoize makes no difference here since the function is called right away. Removing it will simplify the code and might actually slightly improve performance